### DISCUSSION TITLE
add iconsax to components

### DISCUSSION THREAD

**Discussion by MM25Zamanian** [adobe/spectrum-web-components#2046] [Ideas]
Hello
What do you think about an icon component?
The icon pack I have in mind **(if you agree)** is [**iconsax**](https://iconsax.io/)

The preview I have in mind is as follows:
```html
<sp-iconsax size="24" color="fff"></sp-iconsax>
```

I started making this part for myself
I will be happy to help you

**Comment by Westbrook** [comment]
Please take a look at https://opensource.adobe.com/spectrum-web-components/components/icons-workflow/#find-an-icon and see if it solves you're goals here. The likelihood that we choose to support non-Adobe icons at this time is rather low. However, were you not seeing what you wanted, maybe the technique we approach this by would be useful in your own projects.

**Reply by MM25Zamanian** [reply]
Adobe never sees the world 😭😭
